feature-libs/asm/occ/adapters/occ-asm.adapter.ts

```diff
@@ -71,4 +74,29 @@ export class OccAsmAdapter implements AsmAdapter {
       .get<CustomerSearchPage>(url, { headers, params })
       .pipe(this.converterService.pipeable(CUSTOMER_SEARCH_PAGE_NORMALIZER));
   }
+
+  bindCart({ cartId, customerId }: BindCartParams): Observable<unknown> {
```

Comment:
could we just have `bindCart(bindCartParams: BindCartParams)`.
Is there an advantage to destructure the params in the declaration?

Reply:
Destructuring is primarily used for code readability.  
The signature for the `bindCart()` method _is_ `bindCart(options: BindCartParams)`.  

It is functionally equivalent to
```
  bindCart(arg0: BindCartParams): Observable<unknown> {
    const cartId = arg0.cartId;
    const customerId = arg0. customerId
```

The destructuring is just additional syntax to extract particular properties of the parameter object into variables, with the intent of better readability.  Adding a new property to `BindCartParams` would require a text change to the method declaration, but would not result in a change to the signature.